<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastFmSettings</class>
 <widget class="QWidget" name="LastFmSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cb_enable">
     <property name="text">
      <string>Enable LastFM scrobbling</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Authorized User Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_username">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_authentication">
       <property name="text">
        <string>Update Authorization</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>pb_authentication</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
